dev/UndockedRegFreeWinRT/UndockedRegFreeWinRT-AutoInitializer.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Windows.Foundation.UndockedRegFreeWinRTCS
+{
+    internal static class NativeMethods
+    {
+        [DllImport("Microsoft.WindowsAppRuntime.dll", CharSet = CharSet.Unicode, ExactSpelling = true)]
+        internal static extern int UndockedRegFreeWinRT_EnsureIsLoaded();
+    }
+
+    class AutoInitialize
+    {
+        [global::System.Runtime.CompilerServices.ModuleInitializer]
```

Comment:
Warning, this module initializer will execute when folks load the assembly in for reflection and possibly other scenarios. We previously discussed this in another PR  https://github.com/microsoft/WindowsAppSDK/pull/1582#issuecomment-942987325; believe one mitigation was to add a `if (Assembly.GetEntryAssembly() != Assembly.GetExecutingAssembly())` guard.

Reply:
That's interesting. I hadn't seen the earlier comment.

@Scottj1s if you concur I'll do this here _and_ add it to `MddBootstrapAutoInitializer.cs`

Reply:
Added as a proactive change. If problematic I can change or revert.